engine/src/main/scala/cromwell/engine/workflow/lifecycle/MaterializeWorkflowDescriptorActor.scala

```diff
@@ -205,7 +205,7 @@ class MaterializeWorkflowDescriptorActor(serviceRegistryActor: ActorRef,
     val namespaceValidation = validateNamespace(sourceFiles)
     val labelsValidation = validateLabels(sourceFiles.labelsJson)
 
-    (namespaceValidation |@| labelsValidation).tupled flatMap { case (namespace, labels) =>
+    (namespaceValidation, labelsValidation).tupled flatMap { case (namespace, labels) =>
```

Comment:
I kind of liked your `flatMapN` idea if you still think that makes sense

Reply:
I'll give it a go via lenthall. I think I'll do that via a separate PR though